boards.txt

```diff
@@ -1280,6 +1279,18 @@ Genericflight.menu.pnum.AFROFLIGHT_F103CB_12M.build.board=AFROFLIGHT_F103CB_12M
 Genericflight.menu.pnum.AFROFLIGHT_F103CB_12M.build.product_line=STM32F103xB
 Genericflight.menu.pnum.AFROFLIGHT_F103CB_12M.build.variant=AFROFLIGHT_F103CB
 
+# Sparky_V1 board
```

Comment:
To avoid any further issue as there is differentl cortex based, I will remove the default config and set it to sparky:
```patch 
- Genericflight.build.mcu=cortex-m3
- Genericflight.build.series=STM32F1xx
- Genericflight.build.cmsis_lib_gcc=arm_cortexM3l_math
```
Then add them to each sparky `AFROFLIGHT_F103CB` and `AFROFLIGHT_F103CB_12M`

Reply:
Done